rs/tests/driver/src/driver/prometheus_vm.rs

```diff
@@ -253,13 +253,14 @@ for name in replica orchestrator node_exporter; do
   echo '[]' > "{PROMETHEUS_SCRAPING_TARGETS_DIR}/$name.json"
 done
 
+mkdir -p /config/grafana/dashboards
+cp -R /config/grafana/dashboards /var/lib/grafana/
+chown -R grafana:grafana /var/lib/grafana/dashboards
```

Comment:
These two lines don't make sense on the default farm-based PrometheusVM so I would keep them inside the then-branch.